rest/src/test/java/org/envirocar/server/rest/util/OSMTileRendererTests.java

```diff
@@ -29,4 +34,65 @@ public void SaveImageWorks() throws IOException {
 			osmt.saveImage(new BufferedImage(768, 512, BufferedImage.TYPE_INT_RGB), "1235456");
 	}
 	
+	@Test
+	public void zoomLevelTest(){
+		int level=osmt.getZoomLevel(populateCoordinates());
+		assertTrue("Zoom level exceeds limit", 19 >= level);
+		assertTrue("Zoom level lower than limit",  0  <= level);
+	}
+	
+	@Test
+	public void colorCodeTest(){
+		Random r = new Random(); 
```

Comment:
Same as above. You only test the basic range of the color. You should take one or two (e.g. close to minimum and maximum) numbers and assert that they match what you expect for that number. E.g.:

```java
Color color = osmt.getColorCode(45);
assertTrue("wrong red value for color!", color.getRed() == 50) //just an example
assertTrue("wrong green value for color!", color.getGreen() == 50) //just an example
assertTrue("wrong blue value for color!", color.getBlue() == 50) //just an example
```
Also please follow code convention: one whitespace between variable definitions (`color=getColor()`).